data-prepper-plugins/otel-proto-common/src/main/java/org/opensearch/dataprepper/plugins/otel/codec/OTelProtoCodec.java

```diff
@@ -239,22 +223,6 @@ protected Map<String, ResourceSpans> splitResourceSpansByTraceId(final ResourceS
                 }
             }
 
-            if (resourceSpans.getInstrumentationLibrarySpansList().size() > 0) {
```

Comment:
Similar comment as one above: Why is this removed entirely?

Reply:
@dlvenable that API doesn't exist, so it is removed. The ` if (resourceSpans.getScopeSpansList().size() > 0) {` processing scopeSpans.